packages/components/src/tabs/styles.ts

```diff
@@ -181,10 +174,50 @@ export const Tab = styled( Ariakit.Tab )`
 		}
 	}
 
-	[aria-orientation='vertical'] & {
+	[aria-orientation='horizontal'] & {
+		padding: ${ space( 4 ) };
 		min-height: ${ space(
-			10
+			12
 		) }; // Avoid fixed height to allow for long strings that go in multiple lines.
+		height: auto;
```

Comment:
Also in the context of https://github.com/WordPress/gutenberg/issues/65624, the changes from this PR make horizontal tabs with Icons grow from `56px` to `59.5px` tall.

I'm ok with tacking this separately, but let's not forget about it

Reply:
This is fixed now in this PR.